lib/crypto_bottom_sheet/view/crypto_bottom_sheet_view.dart

```diff
@@ -50,42 +51,71 @@ class _CryptoBottomSheetPageState extends State<CryptoBottomSheetPage> {
     );
 
     if (newCryptoAccountName != null &&
-        newCryptoAccountName != cryptoAccountData.name) {
-      await context.read<CryptoBottomSheetCubit>().editCryptoAccount(
-            newAccountName: newCryptoAccountName,
-            index: index,
-          );
+        newCryptoAccountName != cryptoAccountData.name &&
+        newCryptoAccountName.isNotEmpty) {
+      if (accountNameList.contains(newCryptoAccountName)) {
+        AlertMessage.showStringMessage(
+          context: context,
+          message: l10n.sameAccountNameError,
+          messageType: MessageType.error,
+        );
+        return;
+      } else {
+        await context.read<CryptoBottomSheetCubit>().editCryptoAccount(
+              newAccountName: newCryptoAccountName,
+              index: index,
+            );
+      }
     }
   }
 
   Future<void> onAddAccountPressed() async {
-    int index = 0;
-
-    final String? derivePathIndex =
-        await getSecureStorage.get(SecureStorageKeys.derivePathIndex);
-
-    if (derivePathIndex != null && derivePathIndex.isNotEmpty) {
-      index = int.parse(derivePathIndex) + 1;
```

Comment:
before: I used derivative path to assign number to name.
        since derivative path does not increase while importing, issue is occurring

solution: just count the length of the crypto account and assign the number...
white_check_mark
eyes
raised_hands





[10:45](https://talao.slack.com/archives/C02V1G1D9UP/p1658811606180239)
we cannot delete crypto account.. so this solution will work...
[10:45](https://talao.slack.com/archives/C02V1G1D9UP/p1658811627932629)
better solution: keep track of created crypto account and assign the number...